### ISSUE TITLE
No RWSD Config.

### ISSUE THREAD

**Issue by etomoscow** [yandex-research/RuLeanALBERT#4]
I've been looking through your code and in the file data.py I did not find any mentions of the config for the RWSD dataset, while in the article and the leaderboard russiansuperglue.com there is a metric value on this dataset. How were the predictions made if there's no config for that dataset?

**Comment by mryab** [comment]
Hi @etomoscow, sorry for the delayed reply! (turns out I wasn't watching issues for this repository)

In short, a closer look at the leaderboard reveals the answer: the RWSD results for *nearly all* models in top positions are the same. This is not a coincidence: a simple majority vote gives you 0.669 accuracy on RWSD, which shows that it's very hard for most Russian models to outperform this benchmark. In our experiments, we have found early on that all models (including the baseline) quickly collapse to the majority vote solution: hence, we did not publish the config for this dataset, as the same quality could be obtained without lengthy finetuning (in fact, without looking at anything besides the label distribution)

Feel free to follow up with any questions if you have them!
